libs/NfcCoreLib/lib/Nci/phNciNfc_NfceeMgmt.c

```diff
@@ -161,6 +161,10 @@ static NFCSTATUS phNciNfc_CompleteModeSetSequence(void *pContext, NFCSTATUS wSta
              (phNciNfc_IsVersion2x(pNciCtx) &&
                 wStatus != NFCSTATUS_SUCCESS))
         {
+            pNciCtx->IfNtf = pNciCtx->IfModeSetNtf;
+            pNciCtx->IfNtfCtx = pNciCtx->IfNtfModeSetCtx;
```

Comment:
In NCI2.0 this will be triggered before NFCEE_MODE_SET_NTF is received. I don't believe that it is intended behavior.

Reply:
In NCI2.0 this will triggered if the Mode Set Response returns failure.
In success case the call back will be called upon Mode Set Notification.